src/components/Input/Input.css

```diff
@@ -0,0 +1,18 @@
+.canopy__input {
```

Comment:
* The `placeholder` text color should be set to the `dark-alt` hex color
* The input color should be the `dark` hex color

Both can be found in the design token readme
https://github.com/Umbrlla/design-tokens